presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -557,4 +563,17 @@ public TaskManagerConfig setTaskPriorityTracking(TaskPriorityTracking taskPriori
         TASK_FAIR,
         QUERY_FAIR,
     }
+
+    public Duration getInterruptRunawaySplitsTimeout()
+    {
+        return interruptRunawaySplitsTimeout;
+    }
+
+    @Config("task.interrupt-runaway-splits-timeout")
```

Comment:
If we are checking the duration limit in `TaskExecutor` to be at least 1s, we can also add `@MinDuration` here to fail fast.